public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -145,7 +147,12 @@ export const evaluateEveryValidationOptions = (rules: RulerRuleDTO[]): RegisterO
       if (rulesInSameGroupHaveInvalidFor(rules, evaluateEvery).length === 0) {
         return true;
       } else {
-        return `Invalid evaluation interval. Evaluation interval should be smaller or equal to 'For' values for existing rules in this group.`;
+        const rulePendingPeriods = rules.map((rule) => {
+          const { forDuration } = getAlertInfo(rule, evaluateEvery);
+          return safeParsePrometheusDuration(forDuration);
+        });
+        const largestPendingPeriod = Math.min(...rulePendingPeriods);
+        return `Evaluation interval should be smaller or equal to "pending period" values for existing rules in this rule group. Choose a value smaller than or equal to "${formatPrometheusDuration(largestPendingPeriod)}".`;
```

Comment:
We could and I've thought about disabling the buttons and showing a tooltip stating that the option is not available because some rules have a pending period smaller than the interval they want to select.

However I noticed that the current behaviour of showing the error message immediately after selecting the interval is pretty much the same (it explains why the option is not valid and doesn't allow the form to be submitted) but just saves the user a single click in a modal I imagine is not going to be interacted with very often.